### ISSUE TITLE
some question about loss function

### ISSUE THREAD

**Issue by kkdd0717** [open-mmlab/mmaction2#1397]
Hello, first of all, thank you very much for your work.

I want to know where the loss function used in training PoesC3D model is in the program.
If I want to use my own loss function, what should I do?

I hope my problem can be solved, thank you.

**Comment by kkdd0717** [comment]
I tried to find the location of the loss function, but I failed...

**Comment by dreamerlin** [comment]
https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/heads/i3d_head.py#L28

https://github.com/open-mmlab/mmaction2/tree/master/mmaction/models/losses

https://github.com/open-mmlab/mmaction2/blob/master/docs/tutorials/5_new_modules.md#add-new-loss
